src/core/connectivity.jl

```diff
@@ -759,8 +852,9 @@ Dict{Int64, Vector{Int64}} with 12 entries:
 ```
 
 !!! note
-    `get_adjacency_list` cannot be performed for `AllToAllConnectivity`, as in such a connectivity, all qubits are adjacent, 
-    with no upper bound on the number of qubits. A finite list of adjacent qubits thus cannot be constructed. 
+    The `get_adjacency_list` function cannot be used for `AllToAllConnectivity` since this
+    type of connecticty places no upper bound on the number of qubits. A finite list of
```

Comment:
places no upper bound on the number of qubits, and all qubits connect to all others by definition.